scipy/sparse/base.py

```diff
@@ -100,7 +100,26 @@ def get_shape(self):
 
     shape = property(fget=get_shape, fset=set_shape)
 
-    def reshape(self, shape):
+    def reshape(self, shape, order='C'):
+        """
+        Gives a new shape to a sparse matrix without changing its data.
+
+        Parameters
+        ----------
+        shape : length-2 tuple of ints
+            The new shape should be compatible with the original shape.
+        order : 'C', optional
+            This argument is in the signature *solely* for NumPy
+            compatibility reasons for the time being.
```

Comment:
The `pandas` library has dealt with this issue before.  That's where I based the documentation.